### PR TITLE
#6 Components > landing page card

### PR DESCRIPTION
**Pull request by mahmodlte**
Landing page card

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "prop-types": "^15.7.2",
```
Comment (osamaakb) [inline package.json:10]: is this necessary?

File: src/components/Card/LandingPageCard.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import propTypes from 'prop-types';
+const LandingPageCard = ({ image, title }) => {
```
Comment (osamaakb) [inline src/components/Card/LandingPageCard.js:3]: these should be used not only defined

File: src/App.js [hunk 3]
```diff
@@ -1,9 +1,11 @@
 import React from 'react';
+import LandingPageCard from './components/Card/LandingPageCard';
 
 function App() {
   return (
     <div>
       <h1 className="text-3xl text-purple-500">Train To Maintain</h1>
+      <LandingPageCard />
```
Comment (osamaakb) [inline src/App.js:8]: you should pass props here
```suggestion
      <LandingPageCard image={"image URL here"} title={"title gores here"}/>
```

File: src/components/Card/LandingPageCard.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        />
```
Comment (osamaakb) [inline src/components/Card/LandingPageCard.js:10]: no need for this long image `url`

File: package.json (hunk 1 above)
Comment (mahmodlte) [reply package.json:10]: this is for props validation and it is requested my the linting tool

**Pull request closed without merging**